radis/gpu/kernels.cu

```diff
@@ -0,0 +1,175 @@
+
+#include "gpu_cpu_agnostic.h"
+
+extern "C"{
+
+struct initData {
+    float v_min;
+    float v_max;
+    float dv;
+    int N_v;
+    int N_v_FT;
+    int N_x_FT;
+    float dxG;
+    float dxL;
+    int N_lines;
+    float log_c2Mm[16];
+};
+
+
+struct iterData {
+    float p;
+    float log_2p;
+    float hlog_T;
+    float log_rT;
+    float c2T;
+    float N;
+    float l;
+    float slit_FWHM;
+    float log_wG_min;
+    float log_wL_min;
+    int N_G;
+    int N_L;
+    float Q[16];
+};
+
+__device__ __constant__ struct initData init_d;
+__device__ __constant__ struct iterData iter_d;
```

Comment:
Repeated line?

Reply:
they are different structs (init and iter)